docs/conversion/sakai_22_oracle_conversion.sql

```diff
@@ -41,3 +41,324 @@ MODIFIED_DATE TIMESTAMP(6),
 CONSTRAINT commons_like_pk PRIMARY KEY (USER_ID, POST_ID)
 );
 --End SAK-46021
+
+-- SAK-45092
+CREATE TABLE CONV_TOPICS (
+  TOPIC_ID varchar2(36) NOT NULL,
+  ABOUT_REFERENCE varchar2(255) DEFAULT NULL,
+  ALLOW_ANONYMOUS_POSTS raw(1) DEFAULT NULL,
```

Comment:
I'm not sure about using raw here for a boolean type. Oracle doesn't have one, so typically you'll see either a CHAR or NUMBER(1,0) used.